src/social-network-provider/twitter.com/worker/fetch.ts

```diff
@@ -0,0 +1,30 @@
+import { PersonIdentifier, PostIdentifier } from '../../../database/type'
+import { toPostUrl, toProfileUrl } from '../utils/url'
+import tasks from '../../../extension/content-script/tasks'
+import { bioQueryString } from '../utils/selector'
+
+export const fetchPostContent = async (post: PostIdentifier<PersonIdentifier>) => {
+    try {
+        const d = await parseDocument(toPostUrl(post))
+        const content = d.innerText.match(/(🔒.+🔒)/)
```

Comment:
You should take care about the key comes from. If some one commented a key under a normal post, it will be a false-positive and it is dangerous.

Reply:
This also happens at
https://github.com/DimensionDev/Maskbook/blob/1c2f6c4691eaa015298be8667393fdbbfec1f2da/src/social-network-provider/facebook.com/Worker/fetchPostContent.ts#L22
Maybe refactor this first or... whatever, we need a HTML document parser.

Reply:
There is a build-in parser. Checkout http://mdn.io/DOMParser and we're already using it.

Reply:
Okay a TODO will be added.